modules/tinymce/src/plugins/lists/main/ts/core/ListNumbering.ts

```diff
@@ -45,7 +45,7 @@ const isLowercase = (str: string): boolean => /^[a-z]+$/.test(str);
 const isNumeric = (str: string): boolean => /^[0-9]+$/.test(str);
 
 const deduceListType = (start: string): ListType => {
-  if (isNumeric(start)) {
+  if (isNumeric(start) || start === '') {
```

Comment:
This really feels like a hack to be honest, I think instead we should consider adding a `None` list type and branch as needed on that instead.

Reply:
Yup did this.